lib/pagseguro/company.rb

```diff
@@ -0,0 +1,41 @@
+module PagSeguro
+  class Company
+    include Extensions::MassAssignment
+    include Extensions::EnsureType
+
+    attr_accessor :name
+    attr_accessor :display_name
+    attr_accessor :website_url
+
+    attr_reader :address
+    attr_reader :partner
+
+    def address=(address)
+      @address = ensure_type(Address, address)
+    end
+
+    def partner=(partner)
+      @partner = ensure_type(Partner, partner)
+    end
+
+    def phones
+      @phones ||= Phones.new
+    end
+
+    def phones=(_phones=[])
+      _phones.each do |phone|
```

Comment:
I use `each` here because I just want to iterate every phones and attribute it to `phones` and not return an collection, but I know that return it the same way.